dev/PipsPager/PipsPager.cpp

```diff
@@ -127,15 +134,24 @@ winrt::Size PipsPager::GetDesiredPipSize(const winrt::Style& style) {
         {
             if (auto const element = itemTemplate.LoadContent().try_as<winrt::FrameworkElement>())
             {
-                element.Style(style);
+                ApplyStyleToPipAndUpdateOrientation(element, style);
                 element.Measure({ std::numeric_limits<float>::infinity(), std::numeric_limits<float>::infinity() });
                 return element.DesiredSize();
             }
         }
     }
-    /* Extract default sizes and return in case the code above fails */
-    auto pipHeight = unbox_value<double>(ResourceAccessor::ResourceLookup(*this, box_value(c_pipsPagerButtonHeightPropertyName)));
-    auto pipWidth = unbox_value<double>(ResourceAccessor::ResourceLookup(*this, box_value(c_pipsPagerButtonWidthPropertyName)));
+    double pipHeight;
```

Comment:
When is this fallback actually needed?

Reply:
I don't believe we need the fallback if the repeater is there but Orientation can be manually changed before the repeater is loaded and it will call this function so we need a fallback in case that happens :)

Reply:
Why don't we just return {0,0} then? This value is just a guess and not reflective of reality right?

---
In reply to: [599104689](https://github.com/microsoft/microsoft-ui-xaml/pull/4607#discussion_r599104689) [](ancestors = 599104689)

Reply:
yup, we can do that